impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -669,7 +679,12 @@ define(function() {
             yAxisSizeMax: 80, // shouldn't it be: maxHorizontalTextWidth ??
 
             // . dot
-            dot_lineWidth: function() { return this.finished(0); },
+            dot_lineWidth: function() {
+              // show nullShape
+              var v = this.getSize() != null ? 0 : this.delegate();
```

Comment:
The size visual role is optional in both charts. Not sure, but when size is not bound, `getSize()` probably returns `null`; that explains thee check for the size being bound in the Bubble chart, one that seems to also make sense in the HG.